notebooks/query-expansion.ipynb

```diff
@@ -0,0 +1,783 @@
+{
```

Comment:
<p>"user query" instead of "users query"?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/deepset-ai/haystack-cookbook/pull/107/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='0'/>